Integrations/SplunkPy/SplunkPy.py

```diff
@@ -555,18 +555,11 @@ def splunk_parse_raw_command():
 
 def test_module(service):
     if demisto.params().get('isFetch'):
-        t = datetime.utcnow() - timedelta(days=3)
```

Comment:
see the comment about the invalid query.
we don't need really results so one hour is enough.